satpy/readers/ahi_hsd.py

```diff
@@ -221,9 +221,29 @@
 
 
 class AHIHSDFileHandler(BaseFileHandler):
-    """AHI standard format reader."""
+    """AHI standard format reader.
+    The AHI sensor produces data for some pixels outside the Earth disk (i,e:
```

Comment:
Could you add a blank line between the first line and the first paragraph? I think PEP257 suggests this.

Reply:
Done